### PR TITLE
Update eva71-3C-protease.yaml

### PR DESCRIPTION
**Pull request by LizbeK**

### CODE REVIEW

File: data/outputs/TEPs/eva71-3C-protease.yaml [hunk 1]
```diff
@@ -43,9 +56,129 @@ TEP:
...
+      details: |
+        * Vector: pNIC-NHStIIT
+        * Entry clone accession: 
+        * Cell line: 
+        * Tags and additions: His-tag
+        * Construct protein sequence: 
+          MGPSLDFALSLLRRNIRQVQTDQGHFTMLGVRDRLAVLPRHSQPGKTIWVEHKLINILDAVELVDEQGVNLELTLVTLDTNEKFRDITKFIPENISAASDATLVINTEHMPSMFVPVGDVVQYGFLNLSGKPTHRTMMYNFPTKAGQCGGVVTSVGKVIGIHIGGNGRQGFCAGLKRSYFASEQLEHHHHHH
+    - name: A71EV3CPROA-c002 
+      # Brief description of the plasmid
+      description:  Inactive C2A mutant for for enzyme assays.
```
Comment (jchodera) [inline data/outputs/TEPs/eva71-3C-protease.yaml:95]: @LizbeK : Does the inactive protease really work for enzyme assays? Or do you mean biophysical assays?

Comment (jchodera) [reply data/outputs/TEPs/eva71-3C-protease.yaml:95]: And does "C2A" mean residue 2 was mutated?

Comment (LizbeK) [reply data/outputs/TEPs/eva71-3C-protease.yaml]: The inactive protein was made as negative assay controle for the covalent campaign.

Comment (LizbeK) [reply data/outputs/TEPs/eva71-3C-protease.yaml]: Thanks for pointing out terminology error. The active site Cys was mutated to an Ala.

**Pull request merged**